src/contracts/interfaces/IDelegationManager.sol

```diff
@@ -15,6 +15,49 @@ import "./ISignatureUtils.sol";
  * - enabling a staker to undelegate its assets from the operator it is delegated to (performed as part of the withdrawal process, initiated through the StrategyManager)
  */
 interface IDelegationManager is ISignatureUtils {
+    /// @dev Thrown when an account is actively delegated.
+    error ActivelyDelegated();
+    /// @dev Thrown when attempting to execute an action that was not queued.
+    error ActionNotQueued();
```

Comment:
Could perhaps be more explicit instead of action `WithdrawalNotQueued`. Also appreciate the alphabetic ordering here